CHANGELOG.md

```diff
@@ -1,4 +1,8 @@
 # Changelog
+## Unreleased
+### New features
+- New Sudden Appearance model: the sudden_appearance model looks for outliers by finding te sudden appearance of a 
```

Comment:
The sudden appearance of a certain value of a specific field, or multiple fields.